client/components/FlightConfirmation.jsx

```diff
@@ -1,53 +1,40 @@
 import React from 'react';
-
-export default ({ flight }) => {
-
-	const departureDate = flight.scheduledFlights[0].departureTime.split('T');
-	const arrivalDate = flight.scheduledFlights[0].arrivalTime.split('T');
-	const { arrivalAirportFsCode, departureAirportFsCode, carrierFsCode, flightNumber } = flight.scheduledFlights[0];
-	const { airlines } = flight.appendix;
-
-	let arrivalCityName, arrivalAirport, departureCityName, departureAirport, airlineName;
-
-
-	const airport1 = flight.appendix.airports[0];
-	const airport2 = flight.appendix.airports[1];
-
-	if (airport1.iata === arrivalAirportFsCode) {
-		arrivalCityName = airport1.city;
-		arrivalAirport = airport1.name;
-		departureCityName = airport2.city;
-		departureAirport = airport2.name;
-	} else {
-		arrivalCityName = airport2.city;
-		arrivalAirport = airport2.name;
-		departureCityName = airport1.city;
-		departureAirport = airport1.name;
-	}
-
-	for (let i = 0; i < airlines.length; i++) {
-		if (airlines[i].iata === carrierFsCode) {
-			airlineName = airlines[i].name;
-		}
-	}
+import { flightConfirmationConverter } from '../converters/FlightConfirmation';
```

Comment:
This component could still be beautified, but this refactor is more concerned with getting the right data passed in cleanly.